micro_ros_diagnostic_updater/src/micro_ros_diagnostic_updater/micro_ros_diagnostic_updater.c

```diff
@@ -17,21 +17,47 @@
 #include <rcutils/logging_macros.h>
 
 #include <micro_ros_diagnostic_updater/micro_ros_diagnostic_updater.h>
+#include <micro_ros_diagnostic_msgs/msg/micro_ros_diagnostic_status.h>
+
+
+void
+rclc_diagnostic_value_set_int(
+  diagnostic_value_t * kv,
+  int64_t value)
```

Comment:
should be `int32_t`

Reply:
nice catch! I wonder why this didn't raise any pedantic compiler warning about losing precision.